src/openshift/service.ts

```diff
@@ -51,24 +51,18 @@ export class Service extends OpenShiftItem {
     }
 
     static async del(treeItem: OpenShiftObject): Promise<string> {
-        let project: OpenShiftObject;
-        let application: OpenShiftObject;
-        let service: OpenShiftObject = treeItem;
-        if (!service) {
-            project = await vscode.window.showQuickPick(Service.odo.getProjects(), {placeHolder: "From which Project you want to delete Service"});
-            application = await vscode.window.showQuickPick(Service.odo.getApplications(project), {placeHolder: "From which Application you want to delete Service"});
-            if (application) {
-                service = await vscode.window.showQuickPick(Service.odo.getServices(application), {placeHolder: "Select Service to delete"});
-            }
-        }
+        let service: OpenShiftObject;
+        service = await Service.getOpenShiftCmdData(treeItem,
```

Comment:
this should be ```const application = await Service.getOpenShiftCmdData(treeItem,```

Reply:
I am confused. First you call 
``` TypeScript
Service.getOpenShiftCmdData(treeItem, 
    "From which Project you want to delete Service",
    "From which Application you want to delete Service"
);
```
and it passes through treeItem, or returning application istance or null.
Why variable is called `service` then?